app/menu.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import { openExternal } from './utils/open-external';
+import packageJson from '../package.json';
+
+const DOCS_URL = 'https://zepiowallet.com/';
+const REPOSITORY_URL = 'https://github.com/ZcashFoundation/zepio/issues/new';
+
+const menu = [
+  {
+    label: 'Edit',
+    submenu: [
+      { role: 'undo' },
+      { role: 'redo' },
+      { type: 'separator' },
+      { role: 'cut' },
+      { role: 'copy' },
+      { role: 'paste' },
+      { role: 'pasteandmatchstyle' },
+      { role: 'delete' },
+      { role: 'selectall' },
+    ],
+  },
+  {
+    label: 'View',
+    submenu: [
+      { role: 'reload' },
+      { role: 'forcereload' },
+      { type: 'separator' },
+      { role: 'resetzoom' },
+      { role: 'zoomin' },
+      { role: 'zoomout' },
+      { role: 'togglefullscreen' },
+      { type: 'separator' },
+      { role: 'toggledevtools' },
+    ],
+  },
+];
+
+const helpMenu = {
+  role: 'help',
+  submenu: [
+    {
+      label: `v${packageJson.version}`,
+      enabled: false,
+    },
+    { type: 'separator' },
+    {
+      label: 'Help / FAQ',
+      click() {
+        openExternal(DOCS_URL);
+      },
+    },
+    {
+      label: 'Report a Bug',
+      click() {
+        openExternal(REPOSITORY_URL);
+      },
+    },
+  ],
+};
+
+if (process.platform === 'darwin') {
+  menu.unshift({
```

Comment:
couldn't this be done with Ramda/Lodash flow/pipe? 🤔 just for the sake of immutability